src/main/java/com/snack/news/exception/base/NotFoundException.java

```diff
@@ -6,6 +6,11 @@
 @ResponseStatus(value = HttpStatus.NOT_FOUND)
 public abstract class NotFoundException extends RuntimeException {
 	private final static String ERROR_CODE = "NO_RESOURCES";
+	private Long id;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'id' could be replaced by a local variable.](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5174052)